data/korath/korath weapons.txt

```diff
@@ -152,6 +236,82 @@ effect "fire-lance impact"
 	"random angle" 360
 	"velocity scale" 0.
 
+outfit "Blaze-Pike Turret"
+	category "Turrets"
+	cost 1970000
+	thumbnail "outfit/blaze-pike"
+	"mass" 57
+	"outfit space" -54
+	"weapon capacity" -54
```

Comment:
I tend to like decoupled mass/outfits, but that's a good catch in regards to the others.  I think I'll leave this one, but make the Inferno even more heavy - in keeping with the theme that these are more than the sum of their parts, and because slightly slowing down ships that weild them may not be a bad thing.